src/sentry/replays/endpoints/project_replay_accessibility_issues.py

```diff
@@ -86,10 +86,18 @@ def get_result(self, limit, cursor=None):
 
 def request_accessibility_issues(filenames: list[str]) -> Any:
     try:
-        return requests.post(
+        response = requests.post(
             f"{options.get('replay.analyzer_service_url')}/api/0/analyze/accessibility",
             json={"data": {"filenames": filenames}},
-        ).json()
+        )
+
+        content = response.content
+        status_code = response.status_code
+
+        if status_code == 201:
+            return response.json()
+        else:
+            raise ParseError(f"An error occurred: {content}")
```

Comment:
i was slightly confused as usually we return `ParseErrror` to the client, but i see below its being caught. Could this maybe be a `ValueError` or something?

Reply:
Yeah this is all so lazy right now because I can't interact with the service in an easy way.  I think I need to focus on getting the docker image into sentry's devservices.